plugins/node/opentelemetry-instrumentation-redis-4/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@opentelemetry/instrumentation-redis-4",
   "version": "0.39.0",
-  "description": "Automatic OpenTelemetry instrumentation for redis package version 4",
+  "description": "OpenTelemetry instrumentation for `redis` database client for Redis",
```

Comment:
I wonder if keeping the "version 4" clarification here and the "version2  and 3" mention for the "instr-redis" package below is worthwhile.

Reply:
As I wrote in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2196#discussion_r1599361925 This info make sense in the context of one package, but gets confusing when aggregated into a list of packages. As I intend to merge `redis` and `redis-4` in the future, I choose to remove the supported versions to make the text more fluent. 

But I am open to reverting this until the merge to 2 redis packages is introduced

Reply:
For now, I added back the versions for redis, and will target that in a different PR to merge them together.